db/migrate/20171122092336_add_ignored_fact_names_setting.rb

```diff
@@ -0,0 +1,14 @@
+class AddIgnoredFactNamesSetting < ActiveRecord::Migration
+  def up
+    Setting.create!(
+      Setting.set(
+        'excluded_facts',
+        N_("Ignore fact names that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"),
+        Setting[:ignored_interface_identifiers],
+        N_('Ignore fact names with matching identifier'))) if Setting[:ignored_interface_identifiers]
+  end
+
+  def down
+    Setting.where(:name => :excluded_facts).destroy_all
+  end
+end
```

Comment:
Is migration necessary? I thought settings are automatically created during boot time.

Reply:
What is the purpose of the migration?